src/main/java/com/appleaster/task/TaskList.java

```diff
@@ -0,0 +1,169 @@
+package com.appleaster.task;
+
+import com.appleaster.exception.AppleasterException;
```

Comment:
Might have been a mistake of double importing but the import at Line 9 is appropriate. Line 3 should trigger a checkstyle error.